tools/tools.yml

```diff
@@ -0,0 +1,133 @@
+- name: Core
+  packages:
+    - repo: bokeh/bokeh
+      badges: travis, pypi, conda, site
+
+    - repo: matplotlib/matplotlib
+      badges: travis, appveyor, codecov, pypi, conda, site
+
+    - repo: plotly/plotly.py
+      pypi_name: plotly
+      conda_package: plotly
+      conda_channel: conda-forge
+      site: plot.ly
+      badges: circleci, pypi, conda, site
+
+- name: Dashboarding
+  packages:
+
+    - repo: plotly/dash
+      conda_channel: conda-forge
+      site: plot.ly/products/dash/
+      badges: circleci, pypi, conda, site
+
+    - repo: pyviz/panel
+      site: panel.pyviz.org
+      badges: travis, appveyor, codecov, pypi, conda, site
+
+    - repo: QuantStack/voila
+      badges: travis, pypi, conda, rtd
+
+- name: Quick
```

Comment:
I would say that all the ones in quick are high level, and vice versa.  What distinguishes them is maybe "High-Level Shared API" and "High-level"?